pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -752,6 +761,14 @@ func (tmpl *Template) setTemplateObjs(steps []ParallelSteps, dag *DAGTemplate, c
 	tmpl.Suspend = suspend
 }
 
+func (tmpl *Template) GetOutputs() *Outputs {
+	return &tmpl.Outputs
```

Comment:
minor - we can add a nil check in the future to make this nil-safe

Reply:
That's a quick fix, in that case do we want to return an empty `Outputs` or `nil` still if `tmpl.Outputs == nil`? Can do the same fix for the others.

Reply:
I think you can rleae for the moment - I'd return nil